test/E2ETest/Microsoft.Test.E2E.AspNet.OData/QueryComposition/FilterTests.cs

```diff
@@ -181,6 +181,23 @@ public class FilterTests : WebHostTestBase
             }
         }
 
+        public static TheoryDataSet<string, IEnumerable<Movie>> InOperatorData
+        {
+            get
+            {
```

Comment:
What other test cases are there of interest? $filter does not seem to support functions in general. Casting is taken care of from the data sample I updated.